spicepy/_client.py

```diff
@@ -164,8 +163,33 @@ def _api_key(self) -> str:
     def query(self, query: str, **kwargs) -> flight.FlightStreamReader:
         return self._flight.query(query, **kwargs)
 
-    def fire_query(self, query: str, **kwargs) -> flight.FlightStreamReader:
-        return self._firecache.query(query, **kwargs)
+    def refresh_dataset(
+        self, dataset: str, refresh_opts: Optional[Dict[str, str]] = None
+    ) -> Any:
+        if refresh_opts is not None:
```

Comment:
This is probably easier with [dataclasses](https://docs.python.org/3/library/dataclasses.html)?